kumascript/macros/ObsoleteBadge.ejs

```diff
@@ -18,7 +18,7 @@ var titleAttrValue = mdn.localString({
     "id": "This is an obsolete API and is no longer guaranteed to work.",
     "it": "This is an obsolete API and is no longer guaranteed to work.",
     "ja": "これは廃止された API であり、動作は保証されていません。",
-    "ka": "This is an obsolete API and is no longer guaranteed to work.",
+    "ka": "폐기된 API입니다. API의 동작이 보장 되지 않습니다. ",
```

Comment:
Think you ment to change the next line, rather than `ka` which is Georgain

Reply:
```suggestion
    "ko": "폐기된 API입니다. API의 동작이 보장 되지 않습니다. ",
```

Oh I missed this part, thanks. I think the ko below(22line) should be removed.